x-pack/plugins/screenshotting/common/layout.ts

```diff
@@ -69,7 +69,7 @@ export type LayoutParams = Ensure<
  * Supported layout types.
  */
 export const LayoutTypes = {
-  PRESERVE_LAYOUT: 'preserve_layout',
-  PRINT: 'print',
-  CANVAS: 'canvas', // no margins or branding in the layout
+  PRESERVE_LAYOUT: 'preserve_layout' as 'preserve_layout',
+  PRINT: 'print' as 'print',
+  CANVAS: 'canvas' as 'canvas', // no margins or branding in the layout
```

Comment:
Not sure if this will work, but I think we can generalize this with `as const`
```suggestion
  PRESERVE_LAYOUT: 'preserve_layout' as const,
  PRINT: 'print' as const,
  CANVAS: 'canvas' as const, // no margins or branding in the layout
```

Reply:
TIL